webdev/lib/src/command/shared.dart

```diff
@@ -127,13 +125,13 @@ final _defaultWebDirs = const ['web'];
 ///
 /// Throws an [InvalidConfiguration] exception if it can't find at
 /// least one directory.
-Map<String, int> parseDirectoryArgs(List<String> args, {int basePort}) {
-  var result = <String, int>{};
-  basePort ??= 8080;
+Map<String, int?> parseDirectoryArgs(List<String> args, {int? basePort}) {
+  var result = <String, int?>{};
```

Comment:
Can this be non-nullable?

```suggestion
Map<String, int> parseDirectoryArgs(List<String> args, {int? basePort}) {
  var result = <String, int>{};
```

Reply:
Good point, done!